h2o-core/src/main/java/water/fvec/NewChunk.java

```diff
@@ -648,16 +648,21 @@ public void addUUID(UUID uuid) {
 
   // Append a UUID, stored in _ls & _ds
   public void addUUID( long lo, long hi ) {
-    if (C16Chunk.isNA(lo, hi)) throw new IllegalArgumentException("Cannot set illegal UUID value");
-    if( _ms==null || _ds== null || _sparseLen >= _ms.len() )
-      append2slowUUID();
-    _ms.set(_sparseLen,lo);
-    _ds[_sparseLen] = Double.longBitsToDouble(hi);
-    if (_id != null) _id[_sparseLen] = _len;
+    set_impl(_sparseLen, lo, hi);
```

Comment:
The function was added, following the naming of other functions. What's the good name?